tests/www/test_views.py

```diff
@@ -1566,6 +1566,14 @@ def test_tree_view_for_viewer(self):
         resp = self.client.get(url, follow_redirects=True)
         self.check_content_in_response('runme_1', resp)
 
+    def test_gantt_failure_for_viewer(self):
+        url = 'gantt?dag_id=example_bash_operator'
+        self.logout()
+        self.login(username='test_viewer',
+                   password='test_viewer')
+        resp = self.client.get(url, follow_redirects=True)
+        self.check_content_not_in_response('example_bash_operator', resp)
```

Comment:
Why is Viewer not allowed to see this view? It's a read-only view.

Reply:
sorry, wrong unit test. I changed to use /refresh endpoint instead.